README.md

```diff
@@ -226,7 +245,21 @@ public class PeopleController : WebApiController
 }
 ```
 
-*Or if you want to use the Wildcard strategy (which is the default):*
+The previous default strategy (Wildcard) match routes using the asterisk `*` character in the route. **For example:** 
```

Comment:
The previous default strategy (Wildcard) **matches** routes using the asterisk...